app/assets/javascripts/surveys.js

```diff
@@ -443,5 +445,9 @@
   };
 
   root.GOVUK.userSurveys = userSurveys;
+
+  $(document).ready(function() {
+    GOVUK.userSurveys.init();
```

Comment:
Done in a rebase.  Also made it use window.GOVUK instead of root.GOVUK so it's more like other JS files in this project.